point/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "point",
+  "assets": [
+    {
+      "description": "The native governance and staking token of the Point network",
+      "denom_units": [
+        {
+          "denom": "apoint",
+          "exponent": 0
+        },
+        {
+          "denom": "point",
+          "exponent": 18
+        }
+      ],
+      "base": "apoint",
+      "name": "Point",
+      "display": "point",
+      "symbol": "POINT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/pointnetwork/pointnetwork-dashboard/f143abd8e76a652bca407457d179065381274930/assets/point-logo.svg",
+	"png": "https://raw.githubusercontent.com/pointnetwork/pointnetwork/develop/internal/explorer.point/src/assets/pointlogo.png"
+      },
+      "coingecko_id": "point"
```

Comment:
Do you already have this coingecko id?

Reply:
this cgid does not exist, and will cause the checks to fail.

Reply:
@sunnya97 not as yet, should we leave the coingecko id blank for now?